resources/public/cljdoc.js

```diff
@@ -42,6 +43,28 @@ function initSrollIndicator() {
   drawScrollIndicator()
 }
 
+function initToggleRaw() {
+    var toggles = Array.from(document.querySelectorAll(".toggle-raw"))
+
+    function addToggleHandlers() {
+        toggles.forEach((el, idx) => {
+            el.addEventListener("click", function () {
+                var parent = el.parentElement
+                var markdown = parent.querySelector(".markdown")
+                var raw = parent.querySelector(".raw")
+                if (markdown.classList.contains("hide")) {
+                    markdown.classList.remove("hide")
+                    raw.classList.add("hide")
+                } else {
+                    markdown.classList.add("hide")
+                    raw.classList.remove("hide")
+                }
+            })
+        })
+    }
+    addToggleHandlers()
+}
```

Comment:
The file uses 2 spaces for indentation so far which might cause trouble the next time someone auto indents it. Do you think we should switch to 4 spaces or keep 2? I realize 2 is somewhat non-standard in JS 🙂

Reply:
Spacemacs defaulted to 4. I have no js style opinions. I suggest we go with the original style of 2.